lib/ui/painting/engine_layer.cc

```diff
@@ -24,7 +24,7 @@ size_t EngineLayer::GetAllocationSize() {
   // Provide an approximation of the total memory impact of this object to the
   // Dart GC.  The ContainerLayer may hold references to a tree of other layers,
   // which in turn may contain Skia objects.
-  return 3000;
+  return 200000;
```

Comment:
nit: Please add a TODO here linking to https://github.com/flutter/flutter/issues/31498.

Reply:
done